development.md

```diff
@@ -54,6 +54,10 @@ If you install one of our nightly builds to your device, you will not get suppor
 We are using a crowd sourced translation platform to improve the translation and even include new languages.
 If you want to contribute translations you are invited to do so on this platform!
 
+Localization statistics:
+
+[![Crowdin](https://d322cqt584bo4o.cloudfront.net/cgeo/localized.png)](https://crowdin.com/project/cgeo)
```

Comment:
I can't imagine that this host name based reference is stable. But I cannot check myself in the project settings, as I'm not an admin.